### PR TITLE
Issue 23

### LINKED ISSUE #23: No captcha on Jetpack contact form
I manually installed v1.4.2 from the release pages of this repo and enabled it on Jetpack contact form but it does not appear, and the form no longer works.

I'm using Wordpress version 5.4.1 and Jetpack 8.4.2.

Please let me know what types of logs or anything I can provide. I'm not super familiar with Wordpress so I'm not sure where to get useful logs for an issue like this.

### PR DESCRIPTION
**Pull request by kagg-design**
Closes #23

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
-    "description": "hCaptcha WordPress plugin",
-    "type": "project",
-    "license": "MIT",
-    "require": {},
-    "require-dev": {
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
-        "wp-coding-standards/wpcs": "^2.3"
+  "name": "hcaptcha/hcaptcha-wordpress-plugin",
+  "description": "hCaptcha WordPress plugin",
+  "license": "GPL-2.0",
```
Comment (e271828-) [inline composer.json:4]: Please switch back to MIT.

File: .travis.yml [hunk 2]
```diff
@@ -1,43 +1,43 @@
...
+before_script:
   - phpenv rehash
+  - composer install $DEV
 
 script:
   # Search for PHP syntax errors.
   - find -L . -path ./vendor -prune -o -name '*.php' -print0 | xargs -0 -n 1 -P 4 php -l
   # Run PHPCS.
-  - if [[ "$SNIFF" == "1" ]]; then vendor/bin/phpcs . --ignore='*vendor/*'  --standard=WordPress; fi
+  - if [[ "$SNIFF" == "1" ]]; then vendor/bin/phpcs --colors; fi
```
Comment (e271828-) [inline .travis.yml:43]: What's behind all the CI changes? Don't we want `--standard=WordPress` ?

File: composer.json (hunk 1 above)
Comment (kagg-design) [reply composer.json:4]: Plugins in the WordPress repository must be compatible with the GNU General Public License https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/#the-guidelines

Additionally, in the readmt.txt we have 'License: GPLv2 or later'.

File: .travis.yml (hunk 2 above)
Comment (kagg-design) [reply .travis.yml:43]: phpcs.xml file now contains the rule allowing short array syntax (I respect WPCS, but consider long array syntax as overcomplication).

Also, in phpcs.xml added rules to check PHP version compatibility (>=5.6) and WordPress core version compatibility (>=5.1).

File: composer.json (hunk 1 above)
Comment (e271828-) [reply composer.json:4]: MIT is a compatible license, but looks like you're right. "GPLv2 or later, at your option" appears to be correct.

**Pull request merged**